### ISSUE TITLE
Query open orders with websocket

### ISSUE THREAD

**Issue by deployRama** [binance/binance-futures-connector-python#86]
I am not able to query the orders and open positions through a websocket for U futures. I understand that the sdk allows it, but I can't find examples that do it or apparently a script.

Yes, I can query them through RESTApi, but it is having a lot of delay and from the forum they recommended me to try the websocket.

thanks

**Comment by 2pd** [comment]
Thanks for your feedback. It's possible to get the position via websocket when there is account update.

https://binance-docs.github.io/apidocs/futures/en/#event-balance-and-position-update

**Comment by 2pd** [comment]
https://github.com/binance/binance-futures-connector-python/blob/main/examples/websocket/um_futures/user_data.py

is the example to listen to user data stream.

**Comment by deployRama** [comment]
> https://github.com/binance/binance-futures-connector-python/blob/main/examples/websocket/um_futures/user_data.py
> 
> is the example to listen to user data stream.

Thank you very much for your answer. I was having problems with the endpoint that works with the apirest and I wanted to be able to consult via websocket. What you showed me, receiving the events, I had managed to do, but it was not what I was looking for. In these days, I have found out that what I was looking to do is not possible yet. Just listen to events. Thank you very much again.
